static/js/components/EducationForm.js

```diff
@@ -151,22 +183,31 @@ class EducationForm extends ProfileFormFields {
         return <IconButton name="error" onClick={editEntry} />;
       }
     };
-    let dateFormat = date => moment(date).format("MM[/]YYYY");
+    let dateFormat = date => moment(date).format(DASHBOARD_MONTH_FORMAT);
```

Comment:
can we move this function declaration out to be either a class method or a module level function? there's no runtime dependency so we should be OK to do that.